README.md

```diff
@@ -14,19 +14,24 @@
 
 ### Problema a resolver
 
-Hacer la compra es uno de las tareas que tenemos que realizar prácticamente cada día. La forma en la que compramos ha ido evolucionando de la mano de la tecnología: por ejemplo, cuando acudimos a un supermercado, ya es cada vez más frecuente encontrarnos con un método de pago totalmente autónomo, por no hablar por supuesto que ya ni siquiera es necesario acudir presencialmente a supermercados, pues basta con utilizar una app y pedir que te la traigan a tu gusto.
+En el deporte, la disciplina y el entrenamiento son dos máximas que se deben cumplir si se quieren conseguir unos determinados objetivos. Por supuesto, todos queremos evolucionar a una versión mejor de nosotros mismos, y eso es un factor clave en los deportes individuales/grupales, pero a veces nos cuesta seguir una hoja de ruta para ello. En el ámbito del fútbol (en cualquier categoría), gestionar este caso a toda una plantilla puede no ser una tarea tan sencilla, especialmente teniendo características de jugadores tan variadas. Es por tanto que el problema a resolver, puede ser expresada de la siguiente forma: ¿cómo puede un entrenador elegir adecuadamente los ejercicios de entrenamiento para una sesión, mejorando las debilidades y que ayude a conseguir efectos de mejoría en la plantilla?
```

Comment:
Ya ha habido varias propuestas relacionadas con el entrenamiento. También he hablado con entrenadores profesionales sobre el tema. Ninguno de ellos se plantea las rutinas de entrenamiento como un problema, y menos un problema donde haya que plantear algún procesamiento de información más allá de lo trivial (hay que tener cierta progresión, adaptar a la clase, y luego cada entrenador tiene sus rutinas)
Por eso os digo que tengáis conocimiento personal sobre un tema. Si ese es el caso y el resto de las preguntas de la lista (incluyendo que haya datos sobre el tema) se contestan afirmativamente, estoy dispuesto a comprártelo. Si no, desgraciadamente, y como en los otros casos, no parece que sea un problema del que se pueda sacar algo.